docker-compose.yml

```diff
@@ -1,12 +1,12 @@
-version: "3.7"
+version: '3.7'
```

Comment:
Q: Will you be reverting the other changes here other than the version change?

Reply:
Same as above, since using `js-yaml` to edit the version using the .bump-version.js script. everything was getting rewrited.

But already changed it so we only change the version 👍